### PR TITLE
Create spanish readme translation (es-mx)

### PR DESCRIPTION
**Pull request by otelo2**
I've translated the readme to spanish (Méx variation). Please let me know if you need any changes :)
Kindest Regards.

### CODE REVIEW

**Review by github-education — CHANGES_REQUESTED** [review]
**Hi otelo2,**
**Welcome to graduation!**


### I have a few items I need you to take care of before I can merge this PR:

* *Uh Oh! I've found some issues with where you have created your files!* 
	*The required markdown file does not exist, please ensure the file `_data/otelo2/otelo2.md` exists*
*Please ensure all changes are contained within the `_data/otelo2/` directory. Invalid file paths:* 

	* `translations/README.es-mx.md`


Feel free to re-request a review from me and I'll come back and take a look!

File: translations/README.es-mx.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# GitHub Graduation-2022
+
+![2022-github-graduation-social-card-1](/assets/GHG_Blog_1.jpg)
+
+
+Este repositorio contiene el anuario para la **Github Graduation 2022**. Al emitir una pull request al este repositorio, puedes solicitar ser añadido a la Generación Github de 2022.
```
Comment (juanpflores) [inline translations/README.es-mx.md:6]:
```suggestion
Este repositorio contiene el anuario para la **GitHub Graduation 2022**. Al emitir un pull request al este repositorio, puedes solicitar ser añadido a la Generación GitHub de 2022.

```

File: translations/README.es-mx.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# GitHub Graduation-2022
+
+![2022-github-graduation-social-card-1](/assets/GHG_Blog_1.jpg)
+
+
+Este repositorio contiene el anuario para la **Github Graduation 2022**. Al emitir una pull request al este repositorio, puedes solicitar ser añadido a la Generación Github de 2022.
+
+Las primeras 7,500 pull requests que exitosamente se fusionen al repositorio antes del 27 de mayo recibirán una tarjeta comercial personalizada, calcomanías, y una carta por correo.
```
Comment (juanpflores) [inline translations/README.es-mx.md:8]:
```suggestion
Los primeras 7,500 pull requests que exitosamente se fusionen al repositorio antes del 27 de mayo recibirán una tarjeta personalizada, calcomanías, y una carta por correo. 

```

File: translations/README.es-mx.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+Este repositorio contiene el anuario para la **Github Graduation 2022**. Al emitir una pull request al este repositorio, puedes solicitar ser añadido a la Generación Github de 2022.
+
+Las primeras 7,500 pull requests que exitosamente se fusionen al repositorio antes del 27 de mayo recibirán una tarjeta comercial personalizada, calcomanías, y una carta por correo. 
+
+
+## Aviso de privacidad 👀
+Considera que toda la información que añadas a este repositorio estará disponible de forma pública.
+
+- Si no te sientes comodo mostrando tu nombre completo, puedes usar un nombre corto o apodo.
```
Comment (juanpflores) [inline translations/README.es-mx.md:14]:
```suggestion
- Si no te sientes cómodo o cómoda mostrando tu nombre completo, puedes usar un nombre corto o apodo.

```

File: translations/README.es-mx.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+# Quién puede aplicar 📝
```
Comment (juanpflores) [inline translations/README.es-mx.md:16]:
```suggestion
# ¿Quién puede aplicar? 📝

```

File: translations/README.es-mx.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+### Tercero, sube tu Pull Request
+
+Revisa la lista en el formato de pull request para garantizar que tu envío es válido. El equipo de GitHub Education va a revisar tu envío y fusionará tu pull request si todo es correcto. Si no lo es, serás notificado de los cambios necesarios en la sección de comentarios de la pull request.
+
+¿Tienes problemas subiendo tu Pull Request? [Pide ayuda en la Comunidad GitHub](https://github.com/orgs/github-community/discussions/categories/github-education)!
+
+# Historias de Graduación 2022 👩‍🏫👨‍🏫 (opcional)
+¿Buscas más maneras de participar en la Graduación GitHub y la posibilidad de aparecer en nuestras redes sociales?
+
+Queremos escuchar acerca de las cosas increibles que lograste durante tu año académico y sobre cómo GitHub te ayudó a lograr tus metas. Toma un momento para grabar un video o escribir un mensaje y comparte tu historia con nosotros, tus profesores, y tus compañeros de clase
```
Comment (juanpflores) [inline translations/README.es-mx.md:72]:
```suggestion
Queremos escuchar acerca de las cosas increíbles que lograste durante tu año académico y sobre cómo GitHub te ayudó a lograr tus metas. Toma un momento para grabar un video o escribir un mensaje y comparte tu historia con nosotros, tus profesores, y tus compañeros de clase

```

**Review by juanpflores — APPROVED** [review]

**Comment by juanpflores** [comment]
Muchas gracias ✨

**Pull request merged**